advocacy_docs/edb-postgres-ai/migration-etl/data-migration-service/getting_started/installing/linux_x86_64/index.mdx

```diff
@@ -1,48 +1,47 @@
 ---
-title: "Installing EDB DMS Reader on Linux x86 (amd64)"
+title: "Installing EDB Data Migration Service Reader on Linux x86 (amd64)"
 navTitle: "On Linux x86"
 
```

Comment:
If for some reason we cannot get rid of this page as suggested in the previous comments: 

Can we add the following to the frontmatter of this index.mdx?  
```suggestion
indexCards: none

```
That way, we don't have both a Table of Content, and navigation tiles. That way, we can avoid unnecessary repetitions. 
<img width="825" alt="Screenshot 2024-09-24 at 12 39 03 copy" src="https://github.com/user-attachments/assets/89ef123c-2c2d-4d91-85aa-a0890f4068dc">